extensions/cornerstone/src/commandsModule.ts

```diff
@@ -1771,13 +1771,26 @@ function commandsModule({
     },
 
     /**
-     * Sets the fill alpha for inactive segmentations
+     * Sets the fill alpha for inactive segmentations.
+     * If no type is provided, the fill alpha for all types will be set.
      * @param props.type - The type of segmentation
      * @param props.value - The alpha value to set
      */
     setFillAlphaInactiveCommand: ({ type, value }) => {
       const { segmentationService } = servicesManager.services;
-      segmentationService.setStyle({ type }, { fillAlphaInactive: value });
+
+      if (type) {
+        segmentationService.setStyle({ type }, { fillAlphaInactive: value });
+      } else {
+        segmentationService.setStyle(
```

Comment:
This gives the ability for both inactive seg types to be treated uniformly - this makes sense to me because chances are someone who wants to adjust the inactive segs want them ALL to fade away or come into focus together!